server/index.js

```diff
@@ -29,4 +29,6 @@ app.use('/products', require('./routes/products.js'));
 app.use('/reviews', require('./routes/reviews.js'));
 app.use('/qa/questions', require('./routes/questions.js'));
 app.use('/qa/answers', require('./routes/answers.js'));
-app.use('/cart', require('./routes/cart.js'));
\ No newline at end of file
+app.use('/cart', require('./routes/cart.js'));
+
+module.exports = app;
```

Comment:
needed to export here for testing suite (jest?) to be able to import

Reply:
Precisely. This is to remove any conflict of ports. As explained in this (and several others I found) blog post https://zellwk.com/blog/endpoint-testing/